### PR TITLE
Breaking changes list

### PR DESCRIPTION
**Pull request by rrelledge**
[Staged preview](https://docs.redis.com/staging/jira-doc-1882/rs/references/compatibility/breaking-changes/)

### CODE REVIEW

**Review by cmilesb — APPROVED** [review]
Looks good to me!

File: content/rs/references/compatibility/breaking-changes.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+Title: Breaking changes
```
Comment (AlonMagrafta) [inline content/rs/references/compatibility/breaking-changes.md:2]: maybe we should emphasize this is 'redis breaking changes' (as opposed to Redis Enterprise other features)

File: content/rs/references/compatibility/breaking-changes.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+linkTitle: Breaking changes
+description: Potentially breaking changes in Redis Enterprise, introduced by new versions of open source Redis.
```
Comment (AlonMagrafta) [inline content/rs/references/compatibility/breaking-changes.md:4]: that's good :)

File: content/rs/references/compatibility/breaking-changes.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+linkTitle: Breaking changes
+description: Potentially breaking changes in Redis Enterprise, introduced by new versions of open source Redis. 
+weight: 90
+alwaysopen: false
+toc: "false"
+categories: ["RS"]
+aliases: 
+---
+
+When new versions of open source Redis change existing commands, upgrading your database to a new version can potentially break some functionality. Before you upgrade, we recommend you read the provided list of breaking changes that affect Redis Enterprise to help you prepare.
```
Comment (AlonMagrafta) [inline content/rs/references/compatibility/breaking-changes.md:12]:
```suggestion
When new major versions of open source Redis change existing commands, upgrading your database to a new version can potentially break some functionality. Before you upgrade, we recommend you read the provided list of breaking changes that affect Redis Enterprise to help you prepare.
```

**Review by AlonMagrafta — CHANGES_REQUESTED** [review]
Minor comments , but looks good!

**Comment by AlonMagrafta** [comment]
@rrelledge minor comments, looks very good. How do we feel with with another sentence to assist the readers with "how to avoid?' we can also make it an addition later and get this unblocked

**Review by AlonMagrafta — APPROVED** [review]

File: content/rs/references/compatibility/breaking-changes.md (hunk 1 above)
Comment (rrelledge) [reply content/rs/references/compatibility/breaking-changes.md]: Updated the title

**Pull request merged**